query-node/joystream-query-node/scripts/run-dev.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
```

Comment:
This script is for dev purposes only. It start-ups the indexer and the graphql server on the same machine. `reset-dev.sh` drops-creates and then bootstraps the database to simplify the development. Not sure what is the right place for these scripts and if we actually need them to keep in the repo, but thought it may be useful for devs to have them.